tests/jest/unit/assets/dev/js/editor/elements/models/widget.test.js

```diff
@@ -0,0 +1,22 @@
+import { setupMock, freeMock } from './mock/model';
+
+describe( 'Widget', () => {
+	let childModel,
+		widget;
+
+	beforeEach( async () => {
+		childModel = await setupMock();
+
+		widget = new ( await import( 'elementor-elements/models/section' ) ).default;
+	} );
+
+	afterEach( freeMock );
+
+	test( 'isValidChild() -- Sanity', () => {
```

Comment:
Test Nested Widget

Reply:
It will be tested in `dev/nested-elements` PR since the base for Tabs-V2 is in there.
Tabs-v2 are the only widgets that have this feature, and it's not part of the infra.

TODO: Send Commit.